nnvm/src/top/nn/convolution.cc

```diff
@@ -414,6 +414,83 @@ NNVM_REGISTER_OP(_contrib_conv2d_winograd_without_weight_transform)
 
 DMLC_REGISTER_PARAMETER(WinogradConv2DParam);
 
+NNVM_REGISTER_OP(_contrib_conv2d_NCHWc_winograd_weight_transform)
```

Comment:
I think it is better to support NCHWc layout in the original symbol rather than creating a new symbol.